EOS notice (#2842)
signalfx/signalfx-agent: README.md

>ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.

> > # :warning: End of Support (EOS) Notice

Reviewer: ```suggestion
> # :warning: End of Support (EoS) Notice
```

> **The SignalFx Smart Agent has reached End of Support.**
>

> > The [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector) is the successor.
> >
> > To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).
> >
> > Note that this affects the standalone agent; the Smart Agent monitors will be available and supported with the [Smart Agent receiver](https://github.com/signalfx/splunk-otel-collector/blob/main/pkg/receiver/smartagentreceiver/README.md) in the Splunk Distribution of OpenTelemetry Collector.

Reviewer: ```suggestion
> The [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector) is the successor. Smart Agent monitors are available and supported through the [Smart Agent receiver](https://github.com/signalfx/splunk-otel-collector/blob/main/pkg/receiver/smartagentreceiver/README.md) in the Splunk Distribution of OpenTelemetry Collector.
>
> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).
```

>ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.

> # :warning: End of Support (EOS) Notice
> **The SignalFx Smart Agent has reached End of Support.**
>
> The [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector) is the successor.
> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).

> 

Reviewer 2: LGTM, keeping this in mind I suggest we remove the reference to the deprecation notice below.
